README.md

```diff
@@ -320,8 +329,15 @@ Statics parameters
   - Default: `.build/css`
 - `jsPath`: Subdirectory within `staticsRoot` where your JS files are located. By default this folder will not be made public, but is instead meant to store unminified JS source files which will be minified and stored elsewhere when the app is started.
   - Default: `js`
+- `bundledJsPath`: Subdirectory within `staticsRoot` where you would like [browserify](http://browserify.org) to deposit bundled JS files it produces (if you use browserify).
```

Comment:
In my opinion with just these two config options, it isn't clear that bundling is off by default and on when a bundle is specified